core/templates/pages/community-dashboard-page/modal-templates/translation-modal.controller.spec.ts

```diff
@@ -97,7 +98,9 @@ describe('Translation Modal Controller', function() {
     expect($scope.loadingData).toBe(false);
   });
 
-  it('should get suggested text to translated again', function() {
+  it('should suggest more text to be translated when contributor finish' +
+    ' translating text and they would like to continue translating',
```

Comment:
I don't see where the contributor can say "they would like to continue translating"

Reply:
contributor wants to continue translating when `suggestTranslatedText` is called again.